plugins/portlet/pom.xml

```diff
@@ -130,11 +130,7 @@
             <artifactId>spring-test</artifactId>
             <optional>true</optional>
         </dependency>
-        <dependency>
```

Comment:
Maybe we could add some additional infos to the commit to clarify the context

Remove unused spring-webmvc-portlet
Spring 5.x dropped support for spring-webmvc-portlet
https://github.com/spring-projects/spring-framework/wiki/Upgrading-to-Spring-Framework-5.x#dropped-support